packages/helper-process-json/index.js

```diff
@@ -1,4 +1,4 @@
-import jsonPath from 'JSONPath';
+import { JSONPath as jsonPath } from 'jsonpath-plus';
```

Comment:
I did this because eslint complains about `JSONPath` being capitalized and not being used as a constructor.